docs/docs/rest-api/public/api/v2/types/pluginDefinition.raml

```diff
@@ -0,0 +1,14 @@
+#%RAML 1.0 Library
+types:
+  PluginDefinition:
+    type: object
+    properties:
+      id: string
+      plugin: string
+      implementation: string
+      tags?: string[]
```

Comment:
This is interpreted as `Option[Seq[String]]`. What do you think about treating it as an empty string instead?